infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -274,15 +275,15 @@ EOT
       target_host : "api.github.com"
       source_auth_strategy : "oauth2_refresh_token"
       secured_variables : [
-        { name : "ACCESS_TOKEN", writable : true },
+        { name : "PRIVATE_KEY", writable : true },
         { name : "REFRESH_TOKEN", writable : true },
         { name : "OAUTH_REFRESH_TOKEN", writable : true, lockable : true },
-        { name : "CLIENT_ID", writable : false },
-        { name : "CLIENT_SECRET", writable : false }
+        { name : "CLIENT_ID", writable : false }
       ],
       environment_variables : {
-        GRANT_TYPE : "refresh_token"
-        REFRESH_ENDPOINT : "https://github.com/login/oauth/access_token"
+        GRANT_TYPE : "certificate_credentials"
```

Comment:
As a difference from `client_credentials`, this is just to use the JWT obtained without key and directly as authentication header, not as a POST request with a body.